sys/ps/ps.c

```diff
@@ -43,31 +44,31 @@ void ps(void)
     int overall_stacksz = 0, overall_used = 0;
 #endif
 
-    printf("\tpid | "
+    flash_printf(TO_FLASH("\tpid | "
```

Comment:
So this could be just
```suggestion
    printf("\tpid | "
```

now?